modules/sdk/src/main/scala/org/tessellation/sdk/infrastructure/block/processing/BlockAcceptanceManager.scala

```diff
@@ -27,61 +25,60 @@ import org.typelevel.log4cats.slf4j.Slf4jLogger
 
 object BlockAcceptanceManager {
 
-  def make[F[_]: Async: KryoSerializer: SecurityProvider, T <: Transaction: Eq, B <: Block[T]: Eq: Ordering](
-    blockValidator: BlockValidator[F, T, B]
-  ): BlockAcceptanceManager[F, T, B] = make(BlockAcceptanceLogic.make[F, T, B], blockValidator)
+  def make[F[_]: Async: KryoSerializer: SecurityProvider](
+    blockValidator: BlockValidator[F]
+  ): BlockAcceptanceManager[F] = make(BlockAcceptanceLogic.make[F], blockValidator)
 
-  def make[F[_]: Async: KryoSerializer, T <: Transaction: Eq, B <: Block[T]: Eq: Ordering](
-    logic: BlockAcceptanceLogic[F, T, B],
-    blockValidator: BlockValidator[F, T, B]
-  ): BlockAcceptanceManager[F, T, B] =
-    new BlockAcceptanceManager[F, T, B] {
+  def make[F[_]: Async: KryoSerializer](
+    logic: BlockAcceptanceLogic[F],
+    blockValidator: BlockValidator[F]
+  ): BlockAcceptanceManager[F] =
+    new BlockAcceptanceManager[F] {
       private val logger = Slf4jLogger.getLoggerFromClass[F](BlockAcceptanceManager.getClass)
 
       def acceptBlocksIteratively(
-        blocks: List[Signed[B]],
+        blocks: List[Signed[Block]],
         context: BlockAcceptanceContext[F]
-      ): F[BlockAcceptanceResult[B]] = {
+      ): F[BlockAcceptanceResult] = {
 
         def go(
-          initState: BlockAcceptanceState[T, B],
-          toProcess: List[(Signed[B], TxChains[T])]
-        ): F[BlockAcceptanceState[T, B]] =
+          initState: BlockAcceptanceState,
+          toProcess: List[(Signed[Block], TxChains)]
+        ): F[BlockAcceptanceState] =
           for {
-            currState <- toProcess.foldLeftM[F, BlockAcceptanceState[T, B]](initState.copy(awaiting = List.empty)) {
-              (acc, blockAndTxChains) =>
-                blockAndTxChains match {
-                  case (block, txChains) =>
-                    logic
-                      .acceptBlock(block, txChains, context, acc.contextUpdate)
-                      .map {
-                        case (contextUpdate, blockUsages) =>
-                          acc
-                            .focus(_.contextUpdate)
-                            .replace(contextUpdate)
-                            .focus(_.accepted)
-                            .modify((block, blockUsages) :: _)
-                      }
-                      .leftMap {
-                        case reason: BlockRejectionReason =>
-                          acc
-                            .focus(_.rejected)
-                            .modify(_ :+ (block, reason))
-                        case reason: BlockAwaitReason =>
-                          acc
-                            .focus(_.awaiting)
-                            .modify(_ :+ (blockAndTxChains, reason))
-                      }
-                      .merge
-                }
+            currState <- toProcess.foldLeftM[F, BlockAcceptanceState](initState.copy(awaiting = List.empty)) { (acc, blockAndTxChains) =>
```

Comment:
I think the previous formatting was nice.

Reply:
actually it auto formats the code